### DISCUSSION TITLE
lava-nc Slack Workspace & Chat

### DISCUSSION THREAD

**Discussion by mgkwill** [lava-nc/lava-dl#197] [Announcements]
Hi Lava Developers and Users.

**Note: Any support, help or troubleshooting provided is best-effort**

We created a Slack to facilitate user discussions, development tasks and interaction. This workspace enables lava-nc developers and users to collaborate synchronously and create community around the Lava software framework for neuromorphic computing.

Slack Link: https://join.slack.com/t/lava-nc/shared_invite/zt-1v7hm5bw6-6ptVPs0py64aTtSDlRNUQg 
( Link will be updated on a rolling basis)

### lava-nc Slack Rules:
1.    **Do not share _any confidential information about Loihi_, proprietary software or algorithms on an open channel!**
2.    You must respect all users, regardless of your liking towards them. Treat others the way you want to be treated.
3.    Don't send a lot of small messages right after each other. Do not disrupt chat by spamming.
4.    The use of profanity and derogatory language towards any user is prohibited.
5.    Absolutely no NSFW content is permitted.
6.    We do not tolerate any kind of advertisements, whether it be for other communities or workspaces.
7.    You will be asked to change your name or picture if the staff deems it inappropriate.
8.    The Admin will Mute/Kick/Ban per discretion. If you feel mistreated dm an Admin and we will resolve the issue.
9.    The only link sharing permitted is technical content or information that helps resolve a developer's problem.
10.    You may not impersonate or falsely assume the role of an intel or any other company's employee. This includes creating a misleading nickname or false implicit/explicit indications that you are employed by Intel, one of its subsidiaries or any other company in any communications or private direct messages.

**Current Slack Channels:**
- lava-nc
- lava-dl
- lava-dnf
- lava-optimization
- lava-vsa
- lava-docs
- general
- help
- random

**Comment by rileywhite-noblis** [comment]
Hi,

Is could I get an invitation to the slack channel directly? The link only allows those with an @intel email to join directly from the link.
My email is riley.white@nobils.org.

Thanks,
Riley

**Comment by JDRanpariya** [comment]
Hey, Can you please allow me to join the slack channel? Currently it only allows people with intel email id. My email is jay.ranpariya@fau.de

Thanks!!
